tests/kvlds/main.c

```diff
@@ -513,14 +513,14 @@ createmany(struct wire_requestqueue * Q, size_t N)
 		key = kvldskey_create(keybuf, 8);
 		if (proto_kvlds_request_set(Q, key, values[i],
 		    callback_done, NULL))
-			goto err0;
+			goto err1;
```

Comment:
This could be combined with the previous commit, but I figured that this way it was more obvious what was happening.